### ISSUE TITLE
Make use of dunder __len__

### ISSUE THREAD

**Issue by lcandotti** [mdomke/schwifty#72]
The package `schwifty` make extensive use of a property `length`. This makes it hard to:

- Integrate with libraries like django that rely on on the dunder method. (If their's any need to `len(IBAN(...))` we are force to inherit from the class and implement the dunder method __len__ ourselfs).

Was their any reason to create a property length instead of using the dunder ?

**Comment by mdomke** [comment]
I agree that it would be more pythonic to make use of the `__len__` method. I think it was simply not considered at the time of the original implementation, and there is no reason to not add it now.
